src/ResourceManager/ContainerInstance/Commands.ContainerInstance.Test/ScenarioTests/ContainerInstanceTests.ps1

```diff
@@ -23,17 +23,22 @@ function Test-AzureRmContainerGroup
     $location = Get-ProviderLocation "Microsoft.ContainerInstance/ContainerGroups"
     $image = "nginx"
     $osType = "Linux"
```

Comment:
Please add a test for the new parameter set

Reply:
Will the session recording expose storage account key used for volume?

Reply:
Yes - Can you create a test storage account and delete it at the end of the test? This should prevent any security issues caused by exposing the account key.